recipe/build.sh

```diff
@@ -1,14 +1,41 @@
 #!/bin/sh
+
+# Configure git with better timeout settings for network issues
+git config --global http.timeout 600
```

Comment:
Are these time settings to prevent git timeout for git version retrieval things?